src/EasyLogFormatter.php

```diff
@@ -65,11 +80,21 @@ public function formatBatch(array $records)
         return $logBatch;
     }
 
+    /**
+     * @param $record
```

Comment:
Seems like an array, but we have no typehint, if its an array all the time, we can add a typehint

ping @javiereguiluz